security/acme-client/src/opnsense/scripts/OPNsense/AcmeClient/certhelper.php

```diff
@@ -723,6 +723,8 @@ function run_acme_validation($certObj, $valObj, $acctObj)
     // HTTP-01: flush OPNsense port forward rules
     if (($val_method == 'http01') and ((string)$valObj->http_service == 'opnsense')) {
         mwexec('/sbin/pfctl -a acme-client -F all');
```

Comment:
@fichtner All points to this line as the root of the disconnection issue. Am I doing something wrong here with regard to pf anchors wrong?